unstructured_inference/visualize.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*-
+# Copyright (c) Megvii Inc. All rights reserved.
+# Modifications copyright (C) 2023 by Unstructured
+
+import cv2
+import numpy as np
+
+__all__ = ["vis"]
+
+
+def vis(img, boxes, scores, cls_ids, conf=0.5, class_names=None):
```

Comment:
Could you add a doc string to this function and also make the name for explicit? It should be clear from the name what kind of visualization you're getting when you run this.